### PR TITLE
Index the answer objects

### PR DESCRIPTION
**Pull request by safwanrahman**
@pmac r?

### CODE REVIEW

File: kitsune/search/documents.py [hunk 1]
```diff
@@ -1,7 +1,8 @@
+from django.db.models import F
```
Comment (pmac) [inline kitsune/search/documents.py]: I don't see where this is used.

Comment (safwanrahman) [reply kitsune/search/documents.py]: @pmac fixed!

**Comment by safwanrahman** [comment]
@pmac I have fixed the issue. can you review again?

**Comment by pmac** [comment]
Yup. I'll get it in today hopefully.

**Comment by pmac** [comment]
Thanks safwan! I'm going to hopefully land #3416 shortly and then rebase `es_upgrade` on current master so we can keep up and use the new docker stuff.

**Comment by pmac** [comment]
re #3294

**Pull request merged**